model/client4.go

```diff
@@ -2608,7 +2608,14 @@ func (c *Client4) ImportTeam(data []byte, filesize int, importFrom, filename, te
 
 // InviteUsersToTeam invite users by email to the team.
 func (c *Client4) InviteUsersToTeam(teamId string, userEmails []string) (*Response, error) {
-	r, err := c.DoAPIPost(c.teamRoute(teamId)+"/invite/email", ArrayToJSON(userEmails))
+	memberInvite := MemberInvite{
+		Emails: userEmails,
+	}
+	buf, err := json.Marshal(memberInvite)
```

Comment:
We should have another option to invite to team with email + channel to make sure we are not breaking compatabiility

Reply:
compatibility is working. The current member email invite flow keeps working the same, sending a plain email-strings array. Only when the member invite modal is opened from the add member to channel modal (by clicking the 'invite them to the team' new link), the data structure changes to the new memberInvite model structure model.